spread/runner.go

```diff
@@ -277,7 +277,12 @@ func (r *Runner) prepareContent() (err error) {
 		return fmt.Errorf("cannot remove temporary content file: %v", err)
 	}
 
-	args := []string{"c", "--exclude=.spread-reuse.*"}
+	args := []string{
+		"-c",
+		"--exclude=.spread-reuse.*",
+		"--exclude-vcs-ignores",
+		"--exclude-vcs",
```

Comment:
This one may cause problems when one tries to use the information from the actual vcs in a task. A use case could be building a package locally in a prepare and consuming the output of `git describe`.

Reply:
I can drop this. It's not essential for `--exclude-vcs-ignores` from working